src/components/Feeds/FeedPage/FeedPage.jsx

```diff
@@ -113,20 +107,25 @@ const FeedPage = () => {
                         likes={feedLikesData({ feedId: feed._id })}
                         bookmarks={feedUserBookmarksData({ feedId: feed._id })}
                         views={feedViewsData({ feedId: feed._id })}
-                        comments={feedCommentsData({ feedId: feed._id })}
+                        comments={feedData || []}
                         isFeedLikeLoading={isFeedLikeLoading}
                         updateFeedView={true}
                     />
-
-                    <AddFeedComment feedId={feedId} userDetails={userDetails} />
+                    {/* <AddFeedComment feedId={feedId} userDetails={userDetails} /> */}
+                    <ModifyFeedContainer>
```

Comment:
Hey you need this comment in here?

Reply:
Yes, I am not sure, in which way @kabir0x23  would like the add comment to look like
before:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/74410fa5-1fa7-4c83-adb5-573f8317a149)

after:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/a36dec83-1fb3-40f3-907a-d56be80f0c09)